utils/commands.ts

```
@@ -511,7 +511,7 @@ export async function updatePostStatus(id: number, status: string): Promise<void
 export async function readFile(path: string): Promise<string> {
     const cwd = configurations.rootDir;
     const command = wrapPrefix(`sudo cat ${path}`);
-    const result = exec(command, { cwd: cwd, async: false });
+    const result = exec(command, { cwd: cwd, async: false, silent: true });
```

doesn't this mean that we'd silencing other error in future which might be a potential problem ?

---

No this is only not to display the contents of the debug.log in the command line during test run, we are not silencing the error.